static/js/demos.js

```diff
@@ -94,13 +94,8 @@ function update(computer, instructions, num_code_lines, highlighted_registers) {
 
 function populate_code(instructions) {
     let codeHTML = "";
-    for (let i in instructions) {
-        codeHTML += '<p class="symbolic_code" id="symbolic_code';
-        codeHTML += i;
-        codeHTML += '">';
-        codeHTML += instructions[i].toString();
-        codeHTML += "</p>"
-        codeHTML += "\r\n";
+    for (const i in instructions) {
```

Comment:
Thanks! Sometimes it's not super clear whether the template literal syntax is a win for readability, but I wanted to make sure you had the option moving forward.